features/merge_articles.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Merge Articles
+  As a blog administrator
+  In order to share my thoughts with the world
+  I want to be able to merge similar articles on my blog
+
+  Background:
+    Given the blog is set up
+
+  Scenario: Non-admin cannot merge articles
+    Given I am logged in as a contributor
+    And I have two similar articles titled Cats, Dogs
+    And I am on the manage articles page
+    Then I should see "Cats"
+    And I should see "Dogs"
```

Comment:
Do you need to check the articles in order to test that this user won't be able to merge?